pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -323,29 +337,29 @@ func (apimanager *APIManager) setApicastSpecDefaults() bool {
 	changed := false
 	spec := &apimanager.Spec
 
-	defaultApicastManagementAPI := "status"
-	defaultApicastOpenSSLVerify := false
-	defaultApicastResponseCodes := true
-	defaultApicastRegistryURL := "http://apicast-staging:8090/policies"
+	tmpDefaultApicastManagementAPI := defaultApicastManagementAPI
```

Comment:
We create a temporary copy of the constant because below in code we need the address of the variable and in Golang you cannot take it from a constant.

It's not very ideal due to there's duplication by creating the temporary variables here and in the test file.
Another option that I've thought of is creating unexported methods (starting with lowercase) that return a pointer to string and do not define them as constants. The unexported methods would be called from both the code and the tests. What do you think?

Reply:
I think it is ok the way it is now.